sasmodels/modelinfo.py

```diff
@@ -871,18 +890,6 @@ class ModelInfo(object):
     #: arises when the model is constructed using names such as
     #: *sphere*hardsphere* or *cylinder+sphere*.
     composition = None      # type: Optional[Tuple[str, List[ModelInfo]]]
-    #: Name of the control parameter for a variant model such as :ref:`rpa`.
-    #: The *control* parameter should appear in the parameter table, with
-    #: limits defined as *[CASES]*, for case names such as
-    #: *CASES = ["diblock copolymer", "triblock copolymer", ...]*.
-    #: This should give *limits=[[case1, case2, ...]]*, but the
-    #: model loader translates this to *limits=[0, len(CASES)-1]*, and adds
-    #: *choices=CASES* to the :class:`Parameter` definition. Note that
-    #: models can use a list of cases as a parameter without it being a
-    #: control parameter.  Either way, the parameter is sent to the model
-    #: evaluator as *float(choice_num)*, where choices are numbered from 0.
-    #: See also :attr:`hidden`.
-    control = None          # type: str
```

Comment:
After a quick grep, I noticed that convert.py still calls on the control parameter in the function _get_translation_table.

Reply:
Nice catch.  We don't have any automated test cases for the conversion of 3.x to 4.x models, which is how this slipped through.

I copied the logic from sasview_models so it should work as well as it did before.  I didn't move the logic to modelinfo since I expect sasview_models to go away, and translation from 3.x is cruft that shouldn't be embedded in the main code.